### PR TITLE
PORTIFOLLIO CHECKOUT

### PR DESCRIPTION
**Pull request by pmaxy**
**I did the following:**

- run and testing linters
- add html, css files
- update readme file.
- merge the conflicting files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <li><a href="linkedin"><img src="../Portfolio-setup-and-mobile-first/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-setup-and-mobile-first/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-setup-and-mobile-first/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-setup-and-mobile-first/img/chill.svg" alt=""></a></li>
+         </ul> 
+        </div>
+         <img src="../Portfolio-setup-and-mobile-first/imgoverlay/header-shapes mobile@2x.svg" alt="">
+    </header>
+
+
```
Comment (Kamba56) [inline index.html]: - [ ] Please use an indentation of two spaces in all your documents for readability. This is a convention and also [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+         </h3>  
+         <ul class="social-media">
+            <li><a href="twiter"><img src="../Portfolio-setup-and-mobile-first/img/twiter.svg" alt=""></a></li>
+            <li><a href="linkedin"><img src="../Portfolio-setup-and-mobile-first/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-setup-and-mobile-first/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-setup-and-mobile-first/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-setup-and-mobile-first/img/chill.svg" alt=""></a></li>
+         </ul> 
+        </div>
+         <img src="../Portfolio-setup-and-mobile-first/imgoverlay/header-shapes mobile@2x.svg" alt="">
```
Comment (Kamba56) [inline index.html]: - [ ] Please if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. Kindly use CSS to add this image.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                            <li><a href="">About</a></li>
+                            <li><a href="">Context</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </nav>
+    <header class="headline">
```
Comment (Kamba56) [inline index.html]: - [ ] I think the headline section should be a `section` element not a header. The `header` element should contain the `nav` and logo. This is best for HTML semantics. Kindly fix this please.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @pmaxy, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                <li><a href="">Portifolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Context</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    </nav>
```
Comment (pmaxy) [inline index.html:31]:
```suggestion
    <nav class="toolbar">
      <div class="logo"><a href="">My Logo</a></div>
      <div class="menu_main">
        <input type="checkbox" class="toggler">
        <div class="humbuger"><div></div></div>
        <div class="menu">
          <div class="overlay">
            <div>
              <ul>
                <li><a href="">Portifolio</a></li>
                <li><a href="">About</a></li>
                <li><a href="">Context</a></li>
              </ul>
            </div>
          </div>
        </div>
      </div>

    </nav>
    2 spacing
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        </h3>  
+        <ul class="social-media">
+            <li><a href="twiter"><img src="../Portfolio-setup-and-mobile-first/img/twiter.svg" alt=""></a></li>
+            <li><a href="linkedin"><img src="../Portfolio-setup-and-mobile-first/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-setup-and-mobile-first/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-setup-and-mobile-first/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-setup-and-mobile-first/img/chill.svg" alt=""></a></li>
+        </ul> 
+      </div>
+    </section>
```
Comment (pmaxy) [inline index.html:54]:
```suggestion
    
    <nav class="toolbar">
      <div class="logo"><a href="">My Logo</a></div>
      <div class="menu_main">
        <input type="checkbox" class="toggler">
        <div class="humbuger"><div></div></div>
        <div class="menu">
          <div class="overlay">
            <div>
              <ul>
                <li><a href="">Portifolio</a></li>
                <li><a href="">About</a></li>
                <li><a href="">Context</a></li>
              </ul>
            </div>
          </div>
        </div>
      </div>

    </nav>
    <section class="headline">
      <div class="container">
        <h1>I’m Patrick
            glad to see you!</h1>
        <p>I’m a software developer! 
            I can help you build a product ,
            feature or website Look through 
            some of my work and experience! 
            If you like what you see and have 
            a project you need coded, 
            don’t hestiate to contact me.</p>
        <h3>
            <a href="">LET’S CONNECT</a> 
        </h3>  
        <ul class="social-media">
            <li><a href="twiter"><img src="../Portfolio-setup-and-mobile-first/img/twiter.svg" alt=""></a></li>
            <li><a href="linkedin"><img src="../Portfolio-setup-and-mobile-first/img/Linkedin.svg" alt=""></a></li>
            <li><a href="medium"><img src="../Portfolio-setup-and-mobile-first/img/medium.svg" alt=""></a></li>
            <li><a href="github"><img src="../Portfolio-setup-and-mobile-first/img/github.svg" alt=""></a></li>
            <li><a href="chill"><img src="../Portfolio-setup-and-mobile-first/img/chill.svg" alt=""></a></li>
        </ul> 
      </div>
    </section>
```

File: index.html (hunk 4 above)
Comment (pmaxy) [reply index.html:31]: done on section

File: style.css [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  gap: 12px;
+  border-bottom-left-radius: 40px;
+  top: 0;
+  left: calc(50% - (187.5px));
+  overflow: hidden;
+  background: #fff;
+  background-image: url(../Portfolio-setup-and-mobile-first/imgoverlay/header-shapes\ mobile@2x.svg);
+  background-repeat: no-repeat;
+  background-position-x: 0;
+  background-position-y: 100%;
```
Comment (pmaxy) [inline style.css:58]:
```suggestion
  background-image: url(../Portfolio-setup-and-mobile-first/imgoverlay/header-shapes\ mobile@2x.svg);
  background-repeat: no-repeat;
  background-position-x: 0;
  background-position-y: 100%;
```

**Review by frank1738 — APPROVED** [review]
Hi @pmaxy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

- [ ] The microverse image on your readme is broken. Kindly add  the [murple_logo.png](https://github.com/microverseinc/readme-template/blob/master/murple_logo.png) image to your project for it to appear

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**